scripts/vale/vale.js

```diff
@@ -40,16 +46,19 @@ const defaultPatterns = [
 
 const minimistOptions = {
 	string: [
+		'alert',
 		'directory'
 	],
 	boolean: [
 		'save'
 	],
 	alias: {
+		a: 'alert',
 		d: 'directory',
 		s: 'save'
 	},
 	default: {
+		alert: 'warning',
```

Comment:
The default should be the same as set in the `.vale.ini` file in the project. Why do you override the global project setting here?

Reply:
Created new issue for this: https://github.com/ckeditor/ckeditor5/issues/15382